src/layout/burger-menu/burger-blob.svg

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This looks to be unused now?

Reply:
yeah, its removed now. Going to go over and clean up a bit of code!